packages/hostScripts/hostScripts/reboot_required.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Script to check weather a reboot is required or not.
+
+# Exit on error
+set -e
+
+if [ -f /var/run/reboot-required ]; then
+  # print reboot required and pkgs that require reboot
+  if [ -f /var/run/reboot-required.pkgs ]; then
+    # put pkgs in a string comma separated, if is just 1 pkg, do not add comma
+    pkgs=$(cat /var/run/reboot-required.pkgs | tr '\n' ',' | sed 's/,$//')
```

Comment:
are we 100% sure that all machines will have `tr` and `sed`?

Reply:
we cam assume that